packages/mwp-router/src/util/resolve.js

```diff
@@ -1,119 +1,54 @@
 // @flow
```

Comment:
check out the new module in split view - most of the deletions are things that were moved, so you don't need to look at them much.